otx/mpa/seg/incremental/stage.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import numpy as np
+from mmcv import ConfigDict
+
+from otx.mpa.seg.stage import SegStage
+from otx.mpa.utils.config_utils import update_or_add_custom_hook
+from otx.mpa.utils.logger import get_logger
+
+logger = get_logger()
+
+
+class IncrSegStage(SegStage):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def configure_task(self, cfg, training, **kwargs):
+        """Adjust settings for task adaptation"""
```

Comment:
I think this docstring does not match anymore.
Does task adaptation is handle in basic stage?
Then this function is for incremental learning settings